mkdocs.yml

```diff
@@ -83,6 +83,8 @@ plugins:
         - https://mkdocstrings.github.io/objects.inv
         - https://mkdocstrings.github.io/griffe/objects.inv
         options:
+          paths:
+            - src/*
```

Comment:
I only made this change to ensure that this change as a test when running `make docs-serve`. I can remove it, but I wanted to include it initially to show how I tested.

Reply:
Sure, you can remove it now.

Reply:
Done